docs/source/tutorial/custom_components.rst

```diff
@@ -1,71 +1,39 @@
-components, creating visually cohesive pages with ease.
+In this tutorial, we will explore how to create custom **CMS Frontend Components**. These are more
+versatile than :ref:`template_components`, but require some minimal Python coding.
 
-When working with `Tailwind CSS <https://tailwindcss.com>`_, for example, you either create your custom
-frontend components or customize components from providers, e.g. `Tailwind UI <https://tailwindui.com>`_,
-`Flowbite <https://flowbite.com>`_, or the community `Tailwind Components <https://tailwindcomponents.com>`_.
+You create a custom frontend component by subclassing the ``CMSFrontendComponent`` class to 
+declare the form, plus providing a rendering template, and `djangocms-frontend` will take care 
+integrating it automatically.
 
-With django CMS you make your components available to the content editors to simply add them to a page by a
-click **and** frontend developers for use in templates from a single source.
 
-Custom frontend components are more versatile than template components, but require some minimal Python coding.
-Technically, you create a custom frontend component by declaring its change form and rendering template.
+Hero CMS component example
+==========================
 
-Installation
-============
+In this tutorial we will create again a **Hero component** similar to the one in the
+:ref:`previous chapter <template_components>`, but this time via code.
 
-Install ``djangocms-frontend`` and add it to your project as described here: :ref:`built_in_components`.
-
-If you do not use the built-in components, you do not need to add them to your ``INSTALLED_APPS``.
-
-.. code-block:: python
-
-    INSTALLED_APPS = [
-        'easy_thumbnails',
-        'djangocms_link',  # Required if djangocms_frontend.contrib.link is used
-        # Main frontend app - pre-built components not needed
-        'djangocms_frontend',
-    ]
-
-
-Adding Styles and JavaScript
-============================
-
-When building template components, you will need to provide your custom CSS files
-either by adding them to the base templates to load on every page, or by adding a
-django-sekizai block to each component.
-
-Hero component example
-======================
+Directory Structure
+-------------------
 
 ``djangocms-frontend`` will look for custom frontend components in the
-**``cms_components`` module in any of your apps**. This way you can
+**``cms_components`` module in any of your installed apps**. This way you can
 either keep components together in one theme app, or keep them with where
 they are used in different custom apps.
 
-Directory Structure
--------------------
-
-Let's go through this by creating a theme app::
-
-        python manage.py startapp theme
-
-Custom frontend components live in the ``cms_components`` module of any of your apps.
 Ensure your app has the following structure::
 
-    theme/
+    theme_app/
```

Comment:
Changed so that it matches the app name in the previous tutorial - also I like it better because it's clear that it's a django app.